app/app.go

```diff
@@ -8,6 +8,16 @@ import (
 	"net/http"
 )
 
+type App struct {
+	Srv *Server
```

Comment:
I'll be adding some things to `App` that won't go into `Server`. And as discussed in the dev meeting channel, it's likely that the `Server` will be moved out of this struct (and possibly out of the `app` package entirely).

The distinction between "app" and "server" is that the "app" includes all our application logic, while the "server" exposes that application via HTTP. So it's logical that the server would have a dependency on the app, and not the other way around like it's always been. If we change that, I'd like to do it in a smaller follow-up PR.